composer.json

```diff
@@ -22,6 +22,7 @@
     "require-dev": {
         "symfony/phpunit-bridge": "^2.8|^3.0|^4.0|^5.0",
         "doctrine/common": "~2.2",
+        "doctrine/persistence": "^1.3.3",
```

Comment:
if we do this, php should be bumped to >=7.1 above

Reply:
That means Symfony 2.8 support should be dropped as well. Is that something we can do?

Alternatively I can hack something together, like https://github.com/symfony/doctrine-bridge/commit/e3671a2fa2ed4339cea11b6e889639b1e986542e

Reply:
is bumping php to 7.1 acceptable here or do we have to find alternative solution? cc @fabpot